doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -357,13 +357,20 @@ Modem libraries
 
     * Added:
 
+      * Support for setting multiple event callbacks for the default PDP context.
+      * The :c:func:`pdn_default_ctx_cb_dereg` function to deregister a callback for the default PDP context.
```

Comment:
Shouldn't this be documented?

Reply:
Ok, added one line to the conceptual documentation